lib/quadquizaminos/contests.ex

```diff
@@ -0,0 +1,107 @@
+defmodule Quadquizaminos.Contests do
+
+
+  @moduledoc """
+  Inserts and gets data from the database that would be used in different functions.
+  -list of all contests
+  -timer
+  -active contests
+  """
+
+  alias Quadquizaminos.Contest
+  alias Quadquizaminos.Repo
+
+  alias Quadquizaminos.Contest.ContestAgent
+  import Ecto.Query, only: [from: 2]
+
+  @doc """
+  Inserts the created contest in the database
+  """
+  @spec create_contest(map()) :: {:ok, Ecto.Schema.t()}| {:error, Ecto.Changeset.t()}
```

Comment:
`{:ok, Ecto.Schema.t()}` this looks too general and not specific to which schema is used. I would prefer using this instead `{:ok, Contest.t()} / {:ok, %Contest{}}`